server/task.go

```diff
@@ -42,6 +42,10 @@ func (s *Server) registerTaskRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusBadRequest, "Malformatted update task request").SetInternal(err)
 		}
 
+		if taskPatch.EarliestAllowedTs != nil && !s.feature(api.FeatureTaskScheduleTime) {
```

Comment:
Maybe we can extract it with a common template.

xxx is a ${{plan}} feature, please upgrade to access it.

Reply:
Sure, will update it in this PR

Reply:
PTAL